yoda.js

```diff
@@ -0,0 +1,64 @@
+//First time trying to use a mashape api - was not able to get it to work with ajax
+//i still have a lot of practice to do on OOP JS
+
+
+
+$('document').ready(function() {
+  var translate = new Translate();
+
+
+
+var input = ""
+
+function Translate(input) {
+  this.input = input;
+
+  // your code to search the API and return a yoda translate object here
+
+}
+
+
+Translate.prototype = {
+  
+  refresh: function(){
+  	var yodaTranslateURL = "https://yoda.p.mashape.com/yoda?sentence=" + input;
+		return $.ajax({
+	    url: 'yodaTranslateURL', // The URL to the API. You can get this by clicking on "Show CURL example" from an API profile
```

Comment:
I don't think you meant to assign the string 'yodaTranslateURL' to the url property of the ajax call.  Take it small.  Test every step.  Hard code the `input` var and see if the ajax is working before trying to retrieve it from the html.